CHANGES.md

```
@@ -8,6 +8,42 @@ Version 2.3.0
 
 To be released.
 
+### @fedify/fedify
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)